cinema/tests/test_MovieViewSet_api.py

```diff
@@ -0,0 +1,146 @@
+from django.contrib.auth import get_user_model
```

Comment:
ok, you could write your tests in in an existing directory or even had to. But this directory has inappropriate name!